react-frontend/src/components/products/considering.js

```diff
@@ -10,16 +10,16 @@ const cards = [
   <div>
     <p>
       If you are facing an urgent service delivery challenge due to COVID-19,
-      connect with the Digital Response Unit through the{' '}
+      connect with the Digital Delivery Network through the{' '}
       <HrefLink
         href="mailto:exchangelab@gov.bc.ca"
         target="_blank"
         rel="noopener noreferrer"
       >
         Exchange Lab
       </HrefLink>
-      , a community of digital experts who are working together to address
-      urgent challenges.
+      ,a community of digital experts who are working together to address urgent
```

Comment:
need a space between the comma and the a

Reply:
fixed